services/frontend/src/pages/MetaSnapshotPage.tsx

```
@@ -0,0 +1,87 @@
+              {!col.loading && !col.error && col.data.length === 0 && (
+                <div className="col-state muted">No data</div>
+              )}
+
+              {col.data.length > 0 && (
+                <ol className="hero-col-list">
+                  {col.data.map((h, i) => (
+                    <li
+                      key={h.hero_id}
+                      className="hero-col-row"
+                      onClick={() => navigate(`/heroes/${h.hero_id}/${pos}`)}
+                    >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)